docs/0.7/howto/running_echo_service.md

```diff
@@ -0,0 +1,148 @@
+          error_rate:
+              0.05
+          frequency:
+              5
+          jitter:
+              2
+          peer_services:
+              beta
+
+    beta
+        Public Key: <beta_node_public_key>
+        Endpoints:
+            tcps://splinterd-beta:8044
+        Service (echo): b000
+          error_rate:
+              0.05
+          frequency:
+              5
+          jitter:
+              2
+          peer_services:
+              alpha
+```
+
+To get the circuit ID, copy the string following "Circuit:" in the proposal
+output or run the `splinter circuit proposals` command and copy the string in
+the ID column of the output:
+
+```console
+$ splinter circuit proposals –url http://splinterd-beta:8080
+
+ID          NAME MANAGEMENT MEMBERS     COMMENTS PROPOSAL_TYPE
+<circuit_id> echo tutorial alpha;beta -  Create
+```
+
+Vote for the circuit on the beta node using the circuit ID
+
+```console
+$ splinter circuit vote <circuit_id> \
+  --url 'http://splinterd-beta:8080' \
+  --key <path_to_beta_private_key> \
+  --accept
+```
+
+You now have a circuit between node alpha and node beta.
+
+Verify the circuit has been established
+
+```console
+$ splinter circuit list --url 'http://splinterd-alpha:8080'
+
+ID          NAME MANAGEMENT MEMBERS
+abcde-12345 echo tutorial   alpha;beta
+```
+
+The echo services will start running as soon as the circuit is accepted. You
+should see the following logs
+
+```console
+[2022-03-23 14:46:29.859] T[MessageHandlerTaskPool-1] INFO [splinter_echo::service::message_handler] [service:vJKcc-VAuFt::a000] [from:vJKcc-VAuFt::b000] [id:1] received echo response: "test"
+[2022-03-23 14:46:36.589] T[NetworkDispatchLoop] DEBUG [splinter::circuit::handlers::circuit_message] Handle CircuitMessage CIRCUIT_DIRECT_MESSAGE from Peer: Trust ( peer_id: beta ), Local: Trust ( peer_id: alpha ) [76 bytes]
+[2022-03-23 14:46:36.590] T[CircuitDispatchLoop] DEBUG [splinter::circuit::handlers::direct_message] Handle Circuit Direct Message on vJKcc-VAuFt (b000 => a000) [49 bytes]
+[2022-03-23 14:46:36.591] T[MessageHandlerTaskPool-3] INFO [splinter_echo::service::message_handler] [service:vJKcc-VAuFt::a000] [from:vJKcc-VAuFt::b000] [id:1] received echo request, sending echo response: "test"
+[2022-03-23 14:46:36.604] T[NetworkDispatchLoop] DEBUG [splinter::circuit::handlers::circuit_message] Handle CircuitMessage CIRCUIT_DIRECT_MESSAGE from Peer: Trust ( peer_id: beta ), Local: Trust ( peer_id: alpha ) [76 bytes]
+[2022-03-23 14:46:36.608] T[CircuitDispatchLoop] DEBUG [splinter::circuit::handlers::direct_message] Handle Circuit Direct Message on vJKcc-VAuFt (b000 => a000) [49 bytes]
+[2022-03-23 14:46:36.608] T[MessageHandlerTaskPool-3] INFO [splinter_echo::service::message_handler] [service:vJKcc-VAuFt::a000] [from:vJKcc-VAuFt::b000] [id:1] received echo request, sending echo response: "test"
+```
+
+## For More Information
+
+* [Echo Service Design]({ link community/planning/echo_service_design.md})
```

Comment:
This should
`* [Echo Service Design]({% link community/planning/echo_service_design.md%})`

Currently the link doesn't work

Reply:
Fixed